README.md

```diff
@@ -90,7 +90,8 @@ services:
     - /
     plugins:
     - name: grpc-web
-      proto: path/to/hello.proto
+      config:
+        proto: path/to/hello.proto
```

Comment:
is this actually needed?  i don't know the schema by heart, but don't remember ever using a `config:` line like that for any plugin.

Reply:
Yes it is needed, definitively. I got an error when I tried to use it without one.

What might have confused you is that plugins have other fields on the same level as `name` and `config` (like `service` or `protocol`). There's no "automatic movement of unknown fields into the config" feature.

See for example https://docs.konghq.com/hub/kong-inc/basic-auth/#enabling-the-plugin-on-a-service (you might need to click on the "without the database" tab).